app/src/main/java/edu/grinnell/kdic/MainActivity.java

```diff
@@ -65,151 +63,174 @@ public void onServiceConnected(ComponentName className, IBinder service) {
             // if the stream is playing, then stop the notification
             if (radioService.isPlaying()) {
                 radioService.hideNotification();
-                playPauseButton.setImageResource(R.drawable.ic_pause_white_24dp);
+                mPlayPauseButton.setImageResource(R.drawable.ic_pause_white_24dp);
             } else {
-                playPauseButton.setImageResource(R.drawable.ic_play_arrow_white_24dp);
+                mPlayPauseButton.setImageResource(R.drawable.ic_play_arrow_white_24dp);
             }
         }
-
         // Called when the connection with the service disconnects unexpectedly
         public void onServiceDisconnected(ComponentName className) {
-            Log.e(TAG, "onServiceDisconnected");
             boundToRadioService = false;
         }
     };
 
+    /**
+     *  Generate main layout, navigation drawer for the main activity
+     * @param savedInstanceState
+     */
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         // set the layout to use
         setContentView(R.layout.activity_main);
-
-        setupNavigation(); // setup the nav drawer and navigation functionality
+        // setup the nav drawer and navigation functionality
```

Comment:
@LarryAsante I changed and commited a new change in this pull request. Can you take a look at it? Thank you!